src/kava-event-model.js

```diff
@@ -306,6 +306,9 @@ export function getEventModel(eventObj: KavaEvent, model: KavaModel): Object {
   if (model.getApplicationVersion()) {
     commonModel.applicationVersion = model.getApplicationVersion();
   }
+  if (model.userId) {
```

Comment:
keep convention 
```suggestion
  if (model.getUserId()) {
```

Reply:
Why do we have getters? why don't we have prop in object and get the value from there?

Reply:
to support the delegates. if it's props you will need to setup a getter for a prop which is delegated.